hwdb/60-sensor.hwdb

```diff
@@ -328,6 +328,10 @@ sensor:modalias:acpi:SMO8840*:dmi:*:svnLENOVO:pn80XF:pvrLenovoMIIX320*
 # IdeaPad Miix 510
 sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80XE:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+ 
+# IdeaPad Miix 510-12ISK 80U1
+sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80U1:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
```

Comment:
please merge with the previous entry. it's the same product apparently with the same matrix, even though the internal name might have changed? please merge hence and comment why

Reply:
Yes, it's almost the same product as the previous entry. But I am not that sure if all pn80* can use the same matrix.

Reply:
that's not what i am suggesting. But see earlier entries: there can be be multiple match  expressions for the same ACCEL_MOUNT_MATRIX line, and please do that kind of merging. Thanks